src/coprocessor/endpoint.rs

```diff
@@ -252,13 +133,10 @@ impl Host {
         let mut tracker = t.tracker;
         let ranges = req.take_ranges().into_vec();
 
-        let pool_and_ctx_pool = match req.get_context().get_priority() {
-            CommandPri::Low => &mut self.low_priority_pool,
-            CommandPri::High => &mut self.high_priority_pool,
-            CommandPri::Normal => &mut self.pool,
-        };
-        let pool = &pool_and_ctx_pool.pool;
-        tracker.ctx_pool(pool_and_ctx_pool.contexts.clone());
+        let priority = readpool::Priority::from(req.get_context().get_priority());
```

Comment:
I prefer remove package name for types.

Reply:
There are two kind of `Priority` here, one is `kvrpcpb::Priority` and one is `readpool::Priority`. I think it is very necessary to leave package name to avoid confusion.

Reply:
There are two kind of `Priority` here, one is `kvrpcpb::Priority` and one is `readpool::Priority`. I think it is very necessary to leave package name to avoid confusion.

Reply:
Maybe we could rename one of them?

Reply:
They are indeed priorities, just one for internal representation and one for communication. I think it's not a good idea to rename one of them only for saving a package name.